### DISCUSSION TITLE
AppBuilder.WithIcons gone in Avalonia 11?

### DISCUSSION THREAD

**Discussion by arnirichard** [AvaloniaUI/Avalonia#11793] [General]
I can't find anything about WithIcons in 'Breaking Changes'

How do I fix the below problem?

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1061	'AppBuilder' does not contain a definition for 'WithIcons' and no accessible extension method 'WithIcons' accepting a first argument of type 'AppBuilder' could be found (are you missing a using directive or an assembly reference?)	YedaMail	C:\github\yedamail\YedaMail\Program.cs

```
    internal class Program
    {
        // Initialization code. Don't use any Avalonia, third-party APIs or any
        // SynchronizationContext-reliant code before AppMain is called: things aren't initialized
        // yet and stuff might break.
        [STAThread]
        public static void Main(string[] args) => BuildAvaloniaApp()
            .StartWithClassicDesktopLifetime(args);

        // Avalonia configuration, don't remove; also used by visual designer.
        public static AppBuilder BuildAvaloniaApp()
            => AppBuilder.Configure<App>()
                .UsePlatformDetect()
                .LogToTrace()
                .UseReactiveUI()
                .WithIcons(container => container
                    .Register<FontAwesomeIconProvider>());
    }
```

**Comment by timunie** [comment]
It was never a thing that we support I believe. Please check if you have any 3rd party lib for icons and if they supoport Avalonia 11

**Comment by arnirichard** [comment]
Thanks
